src/lib/icons.tsx

```diff
@@ -216,6 +216,34 @@ export const IconSvgs = {
       />
     </svg>
   ),
+  IJPLogo: (
```

Comment:
this should be a webp, since it uses the image tag

Reply:
nice catch!